### DISCUSSION TITLE
How to control Footprints and foot sounds?

### DISCUSSION THREAD

**Discussion by EddieSoloProjects** [Sixze/ALS-Refactored#326] [Q&A]
I am using Ultra Dynamic Sky that allows feet to leave trails in the snow. Problem is the Footprints are Hovering above the snow in the trails. How do I control/remove the footprints and change the sounds when walking?

**Comment by woodencase01** [comment]
For the sound, you can check here: https://github.com/Sixze/ALS-Refactored/discussions/268#discussioncomment-5452204

You'll also find the details about the trace channel and decal for the footprints.
